main/src/main/scala/sbt/EvaluateTask.scala

```diff
@@ -387,7 +388,11 @@ object EvaluateTask {
           val results = x.runKeep(root)(service)
           storeValuesForPrevious(results, state, streams)
           applyResults(results, state, root)
-        } catch { case inc: Incomplete => (state, Inc(inc)) } finally shutdown()
+        } catch {
+          case _: RejectedExecutionException =>
```

Comment:
I would have been happier to translate this into an `Incomplete` in the layer below, but I got lost trying to navigate the code. IntelliJ seems unable to correctly import the SBT project at the moment, which makes my life a bit harder.

Reply:
https://github.com/retronym/xsbt/pull/1